### PR TITLE
add example with SecretMounts to doc

### PR DESCRIPTION
**Pull request by anoipm**
**Description**

Changes proposed in this pull request:

- add example with SecretMounts to doc

**Related issue(s)**
See also #14998

### CODE REVIEW

**Review by pPrecel — COMMENTED** [review]
In general looks ok. 

One question. In other tutorials, we describe how to do things in a few ways (using `kubectl`, `busola` and `kyma-cli`). Would it be valuable to use the same strategy in this tutorial? [Example here](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-serverless/svls-03-expose-function/)

**Comment by anoipm** [comment]
[..]
> One question. In other tutorials, we describe how to do things in a few ways
[...]

I was thinking about it, but decided create example only with Python and kubectl. IMO Node.js not adding nothing new to this example. Kyma CLI don't support SecretMounts, then I can use CLI, but in next step I should edit result form CLI and use kubectl to apply function. I don't know Busola, but it also don't support SecretMounts (I'm right?)

**Review by pPrecel — DISMISSED** [review]

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: Access to Secrets mounted as Volume
+---
+
+This tutorial shows how to use Secrets mounted as volume, with the Serverless Function.
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:5]:
```suggestion
This tutorial shows how to use Secrets mounted as volume with the Serverless Function.
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+We will create a simple Function in Python 3.9. The Function reads data from Secret and returns it.
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:6]:
```suggestion
It's based on a simple Function in Python 3.9. The Function reads data from Secret and returns it.
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    ```
+
+2. Create a Secret:
+
+    ```bash
+    kubectl -n $NAMESPACE create secret generic $SECRET_NAME \
+        --from-literal=$SECRET_DATA_KEY={SECRET_DATA_VALUE}
+    ```
+   
+3. Create your Function with the `secretMounts`:
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:37]:
```suggestion
3. Create your Function with `secretMounts`:
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+              with open(DATA_PATH, "r") as f:
+                data = f.read()
+                return data
+      secretMounts:
+        - secretName: $SECRET_NAME
+          mountPath: $SECRET_MOUNT_PATH
+    EOF
+    ```
+
+   >**NOTE:** Read more about [creating functions](svls-01-create-inline-function.md).
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:64]:
```suggestion
   >**NOTE:** Read more about [creating Functions](svls-01-create-inline-function.md).
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+4. Create ApiRule:
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:66]:
```suggestion
4. Create an APIRule:
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+    This and next steps allow us to test the Function in action.
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:68]:
```suggestion
    The following steps allow you to test the Function in action.
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+            - POST
+            - PUT
+            - DELETE
+      service:
+        name: $FUNCTION_NAME
+        port: 80
+    EOF
+    ```
+
+   >**NOTE:** Read more about [exposing functions](svls-03-expose-function.md).
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:96]:
```suggestion
   >**NOTE:** Read more about [exposing Functions](svls-03-expose-function.md).
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+    curl https://$FUNCTION_NAME.$DOMAIN
+    ```
+
+    You should get `{SECRET_DATA_VALUE}` as a result.
+
+6. Next steps:
+
+    Now you can edit the Secret and see if the Function returns the new value from the Secret.
+    
+    For editing Secret you can use:
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:110]:
```suggestion
    For editing your Secret you can use:
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+    ```bash
+    kubectl -n $NAMESPACE edit secret $SECRET_NAME
+    ```
+   
+    Values used in `data` from Secret should be encoded by `base64`, e.g. like this:
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:115]:
```suggestion
    To encode values used in `data` from the Secret, use `base64`, for example:
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+    ```bash
+    echo -n '{NEW_SECRET_DATA_VALUE}' | base64
+    ```
+
+    Calling the Function again (using `curl`) should return `{NEW_SECRET_DATA_VALUE}`.
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:120]:
```suggestion
    Calling the Function again (using `curl`) must return `{NEW_SECRET_DATA_VALUE}`. 
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+    kubectl -n $NAMESPACE edit secret $SECRET_NAME
+    ```
+   
+    Values used in `data` from Secret should be encoded by `base64`, e.g. like this:
+    ```bash
+    echo -n '{NEW_SECRET_DATA_VALUE}' | base64
+    ```
+
+    Calling the Function again (using `curl`) should return `{NEW_SECRET_DATA_VALUE}`. 
+    Note that Secret propagation may take some time and the call nay return an old value at first.
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:121]:
```suggestion
    Note that the Secret propagation may take some time, and the call may initially return the old value.
```

File: docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+    curl https://$FUNCTION_NAME.$DOMAIN
+    ```
+
+    You should get `{SECRET_DATA_VALUE}` as a result.
+
+6. Next steps:
+
+    Now you can edit the Secret and see if the Function returns the new value from the Secret.
+    
+    For editing your Secret you can use:
```
Comment (grego952) [inline docs/03-tutorials/00-serverless/svls-16-use-secret-mounts.md:110]:
```suggestion
    To edit your Secret, use:
```

**Review by grego952 — APPROVED** [review]

**Pull request merged**